nix/lib/haskell/addHieOutput.nix

```diff
@@ -1,6 +1,4 @@
-{ haskell
-, rsync
-}:
+pkgs:
```

Comment:
wait why was this changed? Now you're passing in an entire nixpkgs instead of the two parts that are necessary..